README.md

```diff
@@ -94,121 +94,157 @@ released, a new version of the JavaScript will be as well.
-  [dep-status]:         https://david-dm.org/swedbankpay/design.swedbankpay.com
-  [devdep-badge]:       https://david-dm.org/swedbankpay/design.swedbankpay.com/dev-status.svg
-  [devdep-status]:      https://david-dm.org/swedbankpay/design.swedbankpay.com?type=dev
-  [og-image]:           https://repository-images.githubusercontent.com/191536903/4ef09b80-53ec-11ea-9f50-8fa24f706dfb
-  [swpdg]:              https://design.swedbankpay.com
-  [nodejs]:             https://nodejs.org/en/
-  [slack]:              https://payex.slack.com/messages/C0L3W8B2S/
-  [react]:              https://reactjs.org/
-  [less]:               http://lesscss.org/
-  [webpack]:            https://webpack.js.org/
-  [semver]:             http://semver.org/
-  [issues]:             https://github.com/swedbankpay/design.swedbankpay.com/issues
-  [erikhallaraaker]:    https://github.com/erikhallaraaker
-  [hmosvoll]:           https://github.com/hmosvoll
-  [VikingTristan]:      https://github.com/VikingTristan
-  [babel]:              https://babeljs.io/
-  [eslint]:             https://eslint.org/
-  [sentry]:             https://sentry.io
-  [stylelint]:          https://stylelint.io/
-  [jest]:               https://facebook.github.io/jest/
-  [enzyme]:             http://airbnb.io/enzyme/
-  [bootstrap]:          http://getbootstrap.com/
-  [materialize]:        https://materializecss.com/
+[version-badge]: https://img.shields.io/badge/Version-4.7.0-blue.svg
+
+[version-tag]: https://github.com/swedbankpay/design.swedbankpay.com/releases/tag/4.7.0
+
+[github-actions-badge]: https://github.com/SwedbankPay/design.swedbankpay.com/workflows/Tag/badge.svg
+
+[codecov-badge]: https://codecov.io/gh/swedbankpay/design.swedbankpay.com/branch/develop/graph/badge.svg
+
+[codecov-status]: https://codecov.io/gh/swedbankpay/design.swedbankpay.com
+
+[dependabot-badge]: https://api.dependabot.com/badges/status?host=github&repo=SwedbankPay/design.swedbankpay.com
+
+[dependabot]: https://dependabot.com
+
+[dep-badge]: https://david-dm.org/swedbankpay/design.swedbankpay.com/status.svg
+
+[dep-status]: https://david-dm.org/swedbankpay/design.swedbankpay.com
+
+[devdep-badge]: https://david-dm.org/swedbankpay/design.swedbankpay.com/dev-status.svg
+
+[devdep-status]: https://david-dm.org/swedbankpay/design.swedbankpay.com?type=dev
+
+[og-image]: https://repository-images.githubusercontent.com/191536903/4ef09b80-53ec-11ea-9f50-8fa24f706dfb
+
+[swpdg]: https://design.swedbankpay.com
+
+[nodejs]: https://nodejs.org/en/
+
+[slack]: https://payex.slack.com/messages/C0L3W8B2S/
+
+[react]: https://reactjs.org/
+
+[less]: http://lesscss.org/
+
+[webpack]: https://webpack.js.org/
+
+[semver]: http://semver.org/
+
+[issues]: https://github.com/swedbankpay/design.swedbankpay.com/issues
+
+[erikhallaraaker]: https://github.com/erikhallaraaker
+
+[hmosvoll]: https://github.com/hmosvoll
+
+[VikingTristan]: https://github.com/VikingTristan
```

Comment:
Codacy found an issue: [[definition-case] Do not use uppercase characters in definition labels](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=5870223)